Description:	Check for file/symlink existence before trying to use it in Java()
Origin:		Debian
Bug-Debian:	http://bugs.debian.org/338232
Forwarded:	http://scons.tigris.org/issues/show_bug.cgi?id=2207

Index: b/engine/SCons/Tool/javac.py
===================================================================
--- a/engine/SCons/Tool/javac.py	2010-01-23 15:11:20.016927869 +0100
+++ b/engine/SCons/Tool/javac.py	2010-01-23 15:11:33.892932280 +0100
@@ -90,6 +90,8 @@
     version = env.get('JAVAVERSION', '1.4')
     full_tlist = []
     for f in slist:
+        if not os.path.exists(f.rfile().get_abspath()):
+            continue
         tlist = []
         source_file_based = True
         pkg_dir = None
